packages/wonder-blocks-dropdown/src/components/combobox.tsx

```diff
@@ -400,6 +410,33 @@ export default function Combobox({
         [selected, setSelected],
     );
 
+    React.useEffect(() => {
+        // Focus on the combobox input when the dropdown is opened.
+        if (openState) {
+            comboboxRef.current?.focus();
+        }
+    }, [openState]);
```

Comment:
**note:** `useEffect` was simplified just to controlling when we need to focus on the input.